keps/sig-autoscaling/20190307-configurable-scale-velocity-for-hpa.md

```diff
@@ -509,7 +513,18 @@ Check the [Default Values][] section for more information about how to determine
 [--horizontal-pod-autoscaler-downscale-stabilization-window]: https://v1-14.docs.kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#algorithm-details
 [DefaultValues]: #default-values
 
-## Graduation Criteria
+## Design Details
+
+### Test Plan
+
+This feature will include the following unit tests to test the following scenarios:
```

Comment:
You could say here the feature *does* include these unit tests.  And provide a permalink to the tests.